### PR TITLE
Add k8s memory dev utils

### PR DESCRIPTION
**Pull request by to-bar**
Adds utils that may be useful for #1908.

### CODE REVIEW

**Review by sk4zuzu — DISMISSED** [review]
:heart_eyes:

File: tools/development/k8s/memory/memory-stress-deployments.yml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        - stress
+        args:
+        - --vm
+        - "1"
+        - --vm-bytes
+        - 1M
+        - --vm-keep
+        - --vm-hang
+        - "1"
+---
```
Comment (atsikham) [inline tools/development/k8s/memory/memory-stress-deployments.yml:216]: Is that line correct or in the correct place?

**Comment by atsikham** [comment]
It would be nice to have a note in some place how to use that.

File: tools/development/k8s/memory/memory-stress-deployments.yml (hunk 1 above)
Comment (to-bar) [reply tools/development/k8s/memory/memory-stress-deployments.yml]: For k8s it is correct but it doesn't comply with yaml standard (i.e. three periods) so I've removed it 👍

**Comment by to-bar** [comment]
> 
> 
> It would be nice to have a note in some place how to use that.

Readme files added 👍

**Review by rafzei — APPROVED** [review]
Nice, I would like to give it a try ASAP 🤟

**Review by atsikham — APPROVED** [review]

**Pull request merged**